config/media-library.php

```diff
@@ -85,6 +85,12 @@
      */
     'conversion_file_namer' => Spatie\MediaLibrary\Conversions\DefaultConversionFileNamer::class,
 
+    /*
+     * This is the class that is responsible for naming files. By default,
+     * it will use the filename of the original and concatenate the conversion name to it.
+     */
+    'file_namer' => Spatie\MediaLibrary\Support\FileNamer\DefaultFileNamer::class,
```

Comment:
To me it's strange that there is a `conversion_file_namer` and a `file_name` wouldn't it be better to merge the concepts into one?